<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preproccessor</class>
 <widget class="QDialog" name="Preproccessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Нагрузки на узлах</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="NodesTable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>39</number>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>84</number>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>F</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Стержни</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="RodsTable">
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>82</number>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>L</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>S</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>E</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>qx</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Допуск. напр</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="AddToRodsButton">
         <property name="text">
          <string>Добавить стержень</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteFromRodsButton">
         <property name="text">
          <string>Удалить стержень</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QCheckBox" name="SealingLeft">
         <property name="text">
          <string>Заделка слева</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="SealingRight">
         <property name="text">
          <string>Заделка справа</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="SaveToFileButton">
       <property name="text">
        <string>Сохранить в файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LoadFromFileButton">
       <property name="text">
        <string>Загрузить из файла</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="ExitButton">
     <property name="text">
      <string>Выйти из препроцессора</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
